kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/core/preparer/InventoryTaskSplitter.java

```diff
@@ -73,6 +77,15 @@ public final class InventoryTaskSplitter {
     
     private final ImporterConfiguration importerConfig;
     
+    private final Map<String, PipelineDataSourceWrapper> actualDataSourceMap;
```

Comment:
1, `actualDataSourceMap` is similar as `sourceDataSource`, could we combine them?

2, Could we just use one `sourceDataSource`? Even it's ShardingSphereDataSource.

Reply:
OK, keep it.